package.json

```diff
@@ -8,11 +8,17 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
+    "test": "jest --no-watchman",
     "clean": "rm -rf dist tsconfig.tsbuildinfo",
     "prepublishOnly": "yarn clean && yarn build"
   },
   "dependencies": {
+    "@ethereumjs/util": "^9.0.0",
+    "ts-jest": "^29.0.1",
```

Comment:
`ts-jest`, `ts-node`, `@types/jest` and `jest` should all be `devDependencies` - they shouldn't be included in the build.